index.js

```diff
@@ -10,25 +10,38 @@ Geoservices.prototype.featureServer = function (req, res) {
   })
 }
 
+/**
+ * Handler for the $namepace/rest/info route. Inspects provider for authentation info and passes any on to the
+ * FeatureServer handler
+ * @param {*} req
+ * @param {*} res
+ */
 Geoservices.prototype.featureServerRestInfo = function (req, res) {
-  FeatureServer.route(req, res)
+  // Inspect provider for an "authInfo" controller; if undefined create a dummy function that returns an empty object
+  let authInfo = this.authInfo || function () { return {} }
+  FeatureServer.route(req, res, authInfo(`${req.protocol}://${req.headers.host}`))
 }
 
 /**
  * Collection of route objects that define geoservices
- * 
- * These routes are bound to the Koop API for each provider. Note that FeatureServer, 
- * FeatureServer/layers, FeatureServer/:layer, and FeatureServer/:layer/:method are found 
+ *
+ * These routes are bound to the Koop API for each provider. Note that FeatureServer,
+ * FeatureServer/layers, FeatureServer/:layer, and FeatureServer/:layer/:method are found
  * in the collection with and without the "$namespace/rest/services/$providerParams" prefix.
- * These prefixed routes have been added due to some clients requiring the "rest/services" 
- * URL fragment in geoservices routes. The $namespace and $providerParams are placeholders 
- * that koop-core replaces with provider specific settings.  
+ * These prefixed routes have been added due to some clients requiring the "rest/services"
+ * URL fragment in geoservices routes. The $namespace and $providerParams are placeholders
+ * that koop-core replaces with provider-specific settings. The 'host' and 'disableIdParam'
+ * properties of a route object will override the properties of the provider - thus, in the
+ * example below '$namespace/rest/info' is not transformed into '$namespace/:host/:id/rest/info
+ * if a provider's 'host' property is true.
  */
 Geoservices.routes = [
   {
     path: '$namespace/rest/info',
     methods: ['get', 'post'],
-    handler: 'featureServerRestInfo'
+    handler: 'featureServerRestInfo',
+    hosts: false,
+    disableIdParam: true
```

Comment:
made a related comment on your PR in koop-core. I'm hoping that we can do this without having `hosts` and `idParam` settings leak beyond where they already are.

Reply:
Yup, I was able to change the approach.  Removed the route-object properties in https://github.com/koopjs/koop-output-geoservices/pull/9/commits/080c3abc0e921aa9561d7085bd561d18b9ba6653.